sockets/sockets.js

```diff
@@ -320,7 +322,57 @@ var actionsObj = {
 					return { message: (Math.floor(Math.random() * 10) + 1).toString(), classStr: "server-text" }
 				}
 			}
-		},
+        },
+        
+        mods: {
+            command: 'mods',
+            help: '/mods: Shows a list of online moderators.',
+            run() {
+                const modUsers = getPlayerUsernamesFromAllSockets().filter((username) => modsArray.includes(username));
+                const message = `Currently online mods: ${modUsers.length > 0 ? modUsers.join(', ') : 'None'}.`;
+                return { message, classStr: "server-text" };
+            }
+        },
+
+        pmmod: {
+            command: 'pmmod',
+            help: '/pmmod: Sends a private message to an online moderator.',
+            run(data, senderSocket) {
+                const { args } = data;
+                // We check if they are spamming, i.e. have sent a PM before the timeout is up
+                const lastPmTime = pmmodCooldowns[senderSocket.id];
+                if (lastPmTime) {
+                    const remaining = new Date() - lastPmTime;
+                    if (remaining < PMMOD_TIMEOUT) return { message: `Please wait ${Math.ceil((PMMOD_TIMEOUT - remaining) / 1000)} seconds before sending another pm!` };
+                }
+                // Checks for various missing fields or errors
+                if (!args[1]) return { message: 'Please specify a mod to message. Type /mods to get a list of online mods.', classStr: 'server-text' };
+                if (!args[2]) return { message: 'Please specify a message to send.', classStr: 'server-text' };
+                if (!modsArray.includes(args[1])) return { message: 'That user is not a mod. You may not private message them.', classStr: 'server-text' };
```

Comment:
Could you please make it say `'<moderator> is not a mod .......... (same as what you had)'`